presto-spi/src/main/java/io/prestosql/spi/eventlistener/QueryStatistics.java

```diff
@@ -26,6 +26,7 @@
     private final Duration queuedTime;
     private final Optional<Duration> waitingTime;
     private final Optional<Duration> analysisTime;
+    private final Optional<Duration> executionTime;
```

Comment:
there are couple of other timing stats in `QueryStats` object. I think we should consistently expose them too (apart from `elapsedTime`)